src/patternfly/components/Divider/examples/Divider.md

```diff
@@ -64,7 +74,7 @@ The divider renders as an `<hr>` by default. It is possible to make the divider
 | Class | Applied to | Outcome |
 | -- | -- | -- |
 | `.pf-c-divider` | `<hr>`, `<li>`, `<div>` | Defines the divider component. |
-| `.pf-m-vertical` | `.pf-c-divider` | Modifies the divider component from horizontal to vertical. This modifier requires that the parent has an explicit or implicit height, or has a flex or grid based layout parent. |
+| `.pf-m-vertical{-on-[breakpoint]}` | `.pf-c-divider` | Modifies the divider component to vertical. This modifier requires that the parent has an explicit or implicit height, or has a flex or grid based layout parent. Modifies the divider back to horizontal at an optional breakpoint. |
 | `.pf-m-inset-{none, sm, md, lg, xl, 2xl, 3xl}{-on-[sm, md, lg, xl, 2xl]}` | `.pf-c-divider` | Modifies divider padding/inset to visually match padding of other adjacent components. |
```

Comment:
Looks like we need to add documentation for the horizontal variant as well.

Reply:
Updated!